.gitignore

```diff
@@ -102,3 +102,9 @@ venv.bak/
 
 # mypy
 .mypy_cache/
+
+# text files
+word_data/
+
+# read me
+README.md
```

Comment:
Why are we ignoring the readme file? did I ask for an updated readme file with instructions?